### DISCUSSION TITLE
AI, LLMs, machine learning, and the Commons

### DISCUSSION THREAD

**Discussion by bjr70** [MESH-Research/knowledge-commons-repository#125] [General]
We've proposed adding a field to indicate that a work was produced using AI with an available notes field to describe its use.

Questions have been raised about our own use of AI, our policies on the use of our data, and the ways in which we should communicate that to users. 

Thoughts? Ideas?

**Comment by bjr70** [comment]
The KCR core team is proposing the following solution: 

Adding the following checkbox to the "People" section of the form: "[ ] Generative AI contributed to the production of this work"

If the checkbox is checked an additional text field would open to allow for the user to provide the details of its use. 

If you have questions or strong feelings on this please respond by 3pm today.

**Comment by tzouris** [comment]
I like the idea of having a checkbox. This is all too new and it would be helpful to embrace the change and let people tell us whether they used an LLM.
